src/chat/components/ConversationItem.tsx

```diff
@@ -35,6 +41,33 @@ export const ConversationItem: React.FC<IConversationItemProps> = ({
     return undefined;
   }, [data.image, data.name]);
 
+  const getInitialsChat = (type: string | undefined): string => {
+    if (!data?.latestMessage || !type) return '';
+
+    const { latestMessage } = data;
+    const { senderId, text } = latestMessage;
+    const isCurrentUser = senderId === userInfo.id;
+
+    const getMessageText = (messageType: string): string => {
+      const senderInfo = `${isCurrentUser ? 'You' : senderId} sent a`;
```

Comment:
I think it better display the name of sender instead of their ID

Reply:
ok I updated